### PR TITLE
add kcc

### PR DESCRIPTION
**Pull request by RyanHubs**

### CODE REVIEW

**Comment by RyanHubs** [comment]
I have read the CLA Document and I hereby sign the CLA

File: gnosis/eth/ethereum_network.py [hunk 1]
```diff
@@ -307,7 +307,7 @@ class EthereumNetwork(Enum):
     FREIGHT_TRUST_NETWORK = 211
     PERMISSION = 222
     SUR_BLOCKCHAIN_NETWORK = 262
-    KCC_MAINET = 321
+    KCC = 321
```
Comment (moisses89) [inline gnosis/eth/ethereum_network.py:310]: I would prefer KCC_MAINNET because there is a KCC_TESTNET.

File: gnosis/eth/multicall.py [hunk 2]
```diff
@@ -50,6 +50,7 @@ class Multicall:
         EthereumNetwork.RINKEBY: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
         EthereumNetwork.ROPSTEN: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
         EthereumNetwork.XDAI: "0x08612d3C4A5Dfe2FaaFaFe6a4ff712C2dC675bF7",
+        EthereumNetwork.KCC: "0x07389Ae9496b188a71dbc11d06081227bAa3DC01",
```
Comment (moisses89) [inline gnosis/eth/multicall.py:53]: Could you verify this address in kcc explorer?

File: gnosis/eth/ethereum_network.py (hunk 1 above)
Comment (Uxio0) [reply gnosis/eth/ethereum_network.py:310]: We use the name you used for chainlist, we are not changing it: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-321.json

**Comment by Uxio0** [comment]
I'm closing this due to inactivity

**Pull request closed without merging**